addon/index.js

```diff
@@ -0,0 +1,30 @@
+const L = window.L || {};
+
+export default L;
```

Comment:
I don't feel enough confidence to use the `default` export yet.
It would make sense for the leaflet package itself, but we may want to use this export for something else later?